pkg/controller/api/radapp.io/v1alpha3/deploymentresource_types.go

```diff
@@ -42,9 +39,6 @@ type DeploymentResourceStatus struct {
 
 	// Phrase indicates the current status of the Deployment Resource.
 	Phrase DeploymentResourcePhrase `json:"phrase,omitempty"`
-
```

Comment:
Was this unused before?

Reply:
I can't tell exactly what this is referring to but if it's the ProviderConfig, yes it's unused now that I'm using the DeploymentsClient instead of the RadiusClient.